modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/ElasticsearchSearchEngine.java

```diff
@@ -144,6 +144,16 @@ public Collection<Long> getIndexedCompanyIds() {
 		return companyIds;
 	}
 
+	@Override
+	public IndexSearcher getIndexSearcher() {
```

Comment:
@W2jx After removing `setXXX` and implementing `getXXX`, the `setXXX` methods and `_xxx` fields in `BaseSearchEngine` are no longer used (even the other implementation doesn't use it). I think we should remove these `getXXX`, `setXXX` and `_xxx` from `BaseSearchEngine`.

Reply:
I will fix it.